### PR TITLE
Quick patch for doc about action arguments.

### PR DESCRIPTION
**Pull request by FrankStain**
As requested in https://groups.google.com/d/msg/python-doit/XfnVbvouoiI/bcdhkTuvBAAJ i've made small description about task arguments.

### CODE REVIEW

File: doc/tasks.rst [hunk 1]
```diff
@@ -168,6 +168,25 @@ check :ref:`tools.LongRunning<tools.LongRunning>` as an example.
 
 
 
+action arguments
+-----------------
+
+You always can pass some arguments into your action in case it's a `python-action`.
```
Comment (schettino72) [inline doc/tasks.rst]: dont need repeat this explanation. It is already said on the section above `python-actions`. You only need to add an example that uses kwargs.

File: doc/tasks.rst [hunk 2]
```diff
@@ -168,6 +168,25 @@ check :ref:`tools.LongRunning<tools.LongRunning>` as an example.
...
+
+For ``args`` may be accepted tuple or list object.
+
+.. literalinclude:: tutorial/task_args.py
+
+For ``kwargs`` only `dict` object may be accepted.
+
+.. literalinclude:: tutorial/task_kwargs.py
+
+Also you may be interested in :ref:`params<task_args.arguments>` and :ref:`getargs<dependencies.getargs>` for passing arguments into actions.
```
Comment (schettino72) [inline doc/tasks.rst]: I try to avoid reference concepts that were not explained yet because first time readers wont be able to understand this.

**Comment by FrankStain** [comment]
Next fix according your suggestions.

**Comment by schettino72** [comment]
rebased, squashed and commited. Thanks. 

I removed the first example you added because there is already an example like that. I also moved it a bit below from where you placed.

**Pull request closed without merging**